include/rmm/detail/aligned.hpp

```diff
@@ -62,6 +62,18 @@ constexpr std::size_t align_up(std::size_t value, std::size_t alignment) noexcep
   return (value + (alignment - 1)) & ~(alignment - 1);
 }
 
+/**
+ * @brief Align up to nearest multiple of the CUDA allocation alignment
+ *
+ * @param[in] v value to align
+ *
+ * @return Return the aligned value, as one would expect
+ */
+constexpr std::size_t align_up_cuda(std::size_t value) noexcept
```

Comment:
This seems unnecessarily specialized over just calling `align_up(value, CUDA_ALLOCATION_ALIGNMENT)` directly.

Reply:
Certainly, but it seems to be used everywhere. Maybe another option is to make `CUDA_ALLOCATION_ALIGNMENT` the default parameter?

Reply:
I don't think it's worthwhile, especially with https://github.com/rapidsai/rmm/pull/883 coming closer to completion and the alignment being no longer fixed.

Reply:
This is really a separate issue from this PR. Filed #917

Reply:
Reverted the alignment changes.